Marble Blast Platinum/platinum/server/scripts/game.cs

```diff
@@ -1437,6 +1437,37 @@
 	return %nearest;
 }
 
+
+function GameConnection::getHighestValueNearestGem(%this) {
+	if (isObject(%this.player)) {
+		return getHighestValueNearestGem(%this.player.getTransform());
```

Comment:
getTransform returns a 7 word string, preferable to keep it to 3 words. either use getWords(getTransform(), 0, 2) or getPosition() instead of getTransform. While it works, would keep it more in line with what the parameter actually expects for code cleanliness.

Reply:
I'm a bit confused by this suggestion, because this code here is actually a copypasta of existing code. You can see the original file has:
```
function GameConnection::getNearestGem(%this) {
	if (isObject(%this.player)) {
		return getNearestGem(%this.player.getTransform());
	}
	return getNearestGem("0 0 0");
}
```

So why would this code be different?